projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -995,12 +995,7 @@ public function get_filtered_connection_data( $selected_post_id = null ) {
 				$connection_list[] = array_merge(
 					$connection,
 					array(
-						'enabled'    => $enabled,
-						// Deprecated fields.
-						'id'         => $connection_id,
-						'unique_id'  => $unique_id,
-						'done'       => $done,
-						'toggleable' => $toggleable,
```

Comment:
Looks like that means we can delete the definition of $toggleable above

Reply:
Yes, I nuked it in a subsequent commit.